### PR TITLE
Fixes 264: Image picker Activity leaking window

### PR DESCRIPTION
**Pull request by rohanagarwal94**
Fixes issue #264

### CODE REVIEW

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @rohanagarwal94 we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/rohanagarwal94/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: app/src/main/java/vn/mbm/phimp/me/ImagePickerActivity.java [hunk 1]
```diff
@@ -320,6 +329,8 @@ protected void onPreExecute() {
             showProgressDialog("Loading..");
         }
 
+
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/ImagePickerActivity.java]: Unwanted blank lines.

**Comment by rohanagarwal94** [comment]
@Abhi2424shek made the changes, please review.

**Comment by mariobehling** [comment]
Ok, let's iterate from here. Thank you for your contribution!

**Pull request merged**